OpenRA.Mods.Common/Activities/FindResources.cs

```diff
@@ -103,7 +103,7 @@ public override Activity Tick(Actor self)
 				foreach (var n in notify)
 					n.MovingToResources(self, closestHarvestablePosition.Value, this);
 
-				return ActivityUtils.SequenceActivities(mobile.MoveTo(closestHarvestablePosition.Value, 1), new HarvestResource(self), this);
+				return ActivityUtils.SequenceActivities(mobile.MoveTo(closestHarvestablePosition.Value, 2), new HarvestResource(self), this);
```

Comment:
The real issue is that `closestHarvestablePosition` is demonstrably wrong (if it is blocked then it is *not* harvestable).  We need a fix to make that calculation work correctly rather than a bandaid to try minimize the damage (which may introduce other unexpected effects in the process).